src/main/kotlin/mate/academy/LetterRemover.kt

```diff
@@ -1,5 +1,10 @@
 package mate.academy
 
-class LetterRemover {
-    // implement ...
+object LetterRemover {
+    @JvmStatic
+    fun removeLetter(input: String): CharArray {
+        return input.lowercase()
+            .map { if (it == 'r') '!' else it }
```

Comment:
Interesting solution, however please check this link
https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.text/replace.html